app/assets/stylesheets/spree/frontend/solidus_oxxo_pay.css

```diff
@@ -76,7 +76,7 @@
 	line-height: 14px;
 }
 
-.opps-reference {
+.opps-reference-container {
 	margin-top: 14px;
```

Comment:
I think that the indentation is not the correct here

Reply:
fixed, thanks